src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -267,67 +265,26 @@ def from_string(cls, interval: str) -> 'Interval':
         return Interval(minutes)
 
 
-class StartTime:
-
-    def __init__(self,
-                 hour: int,
-                 minute: int,
-                 tzinfo: Optional[datetime.tzinfo]) -> None:
-        self.time = datetime.time(hour, minute, tzinfo=tzinfo)
-        self.minutes = self.time.hour * 60 + self.time.minute
-        if self.time.tzinfo:
-            utcoffset = cast(datetime.timedelta, self.time.utcoffset())
-            self.minutes += int(utcoffset.seconds / 60)
-
-    def __eq__(self, start_time: Any) -> bool:
-        return self.minutes == start_time.minutes
-
-    def __hash__(self) -> int:
-        return hash(self.minutes)
-
-    def to_string(self) -> str:
-        return self.time.isoformat()
-
-    @classmethod
-    def from_string(cls, start_time: Optional[str]) -> Optional['StartTime']:
-        if not start_time:
-            return None
-
-        try:
-            t = parse(start_time).timetz()
-        except ValueError as e:
-            raise ValueError("Invalid start time {}: {}".format(start_time, e))
-
-        return StartTime(t.hour, t.minute, tzinfo=t.tzinfo)
-
-
 class Schedule:
 
     def __init__(self, name: str) -> None:
         self.name = name
-        self.items: Set[Tuple[Interval, Optional[StartTime]]] = set()
+        self.intervals: Set[Interval] = set()
```

Comment:
Do you think it might be worth it to highlight the change from `items` to `intervals` in the CLI structured output in the release note?